WidgetInspector.cpp

```diff
@@ -115,6 +120,33 @@ void WidgetInspector::copyDebuggerReference()
 #endif
 }
 
+void WidgetInspector::flashWidget()
+{
+	ObjectProxy::Pointer object = m_objectInspector->object();
+	if (!object)
+	{
+		return;
+	}
+
+	QString origStyle;
+	for (const auto &property : object->properties())
+	{
+		if (property.name == "styleSheet")
+		{
+			origStyle = property.value.toString();
+			break;
+		}
+	}
+	QString newStyle = origStyle + " background-color: red; color: limegreen;";
+
+	object->writeProperty("styleSheet", newStyle);
+
+	QTimer::singleShot(500, [=]() { object->writeProperty("styleSheet", origStyle); });
+	QTimer::singleShot(1000, [=]() { object->writeProperty("styleSheet", newStyle); });
+	QTimer::singleShot(1500, [=]() { object->writeProperty("styleSheet", origStyle); });
```

Comment:
This project ostensibly still supports Qt 4, which doesn't support this syntax for `QTimer::singleShot`. I think enough time has passed that Qt 4 support could be removed, but until that change is made, I added a Qt version check around this call.